src/subdomains/supporting/notification/enums/index.ts

```diff
@@ -8,6 +8,7 @@ export enum MailType {
   ERROR_MONITORING = 'ErrorMonitoring',
   USER = 'User',
   PERSONAL = 'Personal',
+  INTERNAL = 'Internal',
```

Comment:
Is INTERNAL the right name? 😉  KYC_SUPPORT and ERROR_MONITORING are also internal.